satpy/readers/__init__.py

```diff
@@ -370,7 +366,7 @@ def get_valid_reader_names(reader):
     return new_readers
 
 
-def available_readers(as_dict=False):
+def available_readers(as_dict=False, yaml_loader=UnsafeLoader):
```

Comment:
Can you add `yaml_loader` to the docstring please.

Reply:
@djhoese done.